hub-detect/src/test/groovy/com/blackducksoftware/integration/hub/detect/bomtool/clang/ClangExtractorTest.java

```diff
@@ -189,4 +192,92 @@ public void testMultipleCommandsDependenciesPackages() throws IOException, Execu
         }
     }
 
+    @Test
+    public void testJsonWithArgumentsNotCommand() throws IOException, ExecutableRunnerException {
```

Comment:
The size of the test is a concern.

Reply:
Yes, that test has a lot of code in common with the previous one. Definitely a potential for refactoring.